src/Repository.js

```diff
@@ -16,7 +16,6 @@ export default class Repository {
     this.rootPath = path.resolve(GitUtilities.getTopLevelDirectory());
     this.lernaJsonLocation = path.join(this.rootPath, "lerna.json");
     this.packageJsonLocation = path.join(this.rootPath, "package.json");
-    this.packagesLocation = path.join(this.rootPath, "packages"); // TODO: Kill this.
```

Comment:
I agree we need to remove this unused property, but I'm not sure this PR is the place for that.